CHANGES_NEXT_RELEASE

```diff
@@ -1,2 +1,3 @@
 - Fix: crash when using null character in some NGSIv1 operations
-- Hardening: NULL control in some strdup and calloc cases (very unlikely, but theoretically possible)
+- Hardening: NULL control in some strdup and calloc cases (very unlikely, but theoretically possible) (#3578)
+- Hardening: avoid crash in the unrare case MongoDB get databases operations fails in csubs cache refresh logic (#3456, partly)
```

Comment:
unrare?
Not sure that is a word.
If it was, it'd mean _normal_ :)
Did you mean **unlikely**?

Reply:
Fixed in 86f9163a49f95319ba8f955171ff259712bfa237